src/cgame/cg_predict.cpp

```diff
@@ -194,14 +194,12 @@ static void CG_ClipMoveToEntities( const vec3_t start, const vec3_t mins,
 			break;
 
 		case traceType_t::TT_BISPHERE:
-			ASSERT(maxs != nullptr);
-			ASSERT(mins != nullptr);
```

Comment:
The pointers have to be non-null because they are dereferenced in the next line.

Reply:
+1

Reply:
I mean, yes, I agree on that, but what's the difference between crashing and crashing? Like the assert is not revealing a programming error that would have been noticed anyway?